docs/api/conventions.md

```diff
@@ -191,9 +191,9 @@ app/
 
 The file in `app/root.tsx` is your root layout, or "root route" (very sorry for those of you who pronounce those words the same way!). It works just like all other routes:
 
-- You can export a [`loader`](#loader), [`action`](#action), [`meta`](#meta), [`headers`](#headers), or [`links`](#links) function
-- You can export an [`ErrorBoundary`](#errorboundary) or [`CatchBoundary`](#catchboundary)
-- Your default export is the layout component that renders the rest of your app in an [`<Outlet />`](https://reactrouter.com/docs/en/v6/api#outlet)
+- You can export a [`loader`][3], [`action`][4], [`meta`][5], [`headers`][6], or [`links`][7] function
+- You can export an [`ErrorBoundary`][8] or [`CatchBoundary`][9]
+- Your default export is the layout component that renders the rest of your app in an [`<Outlet />`][reactrouter-1]
```

Comment:
Maybe we could combine some of the logic of the anchor tags I describe in the previous comment with the domain logic of external links. So this could be:

```suggestion
- Your default export is the layout component that renders the rest of your app in an [`<Outlet />`][reactrouter-outlet]
```

Then maybe we could reduce the number of times we need a post-fix number.

Also, on that note, let's not add a post-fix for the first link. So if there's a duplicate we'd only get a `-2` but we wouldn't get a `-1`.